frontend/public/components/cluster-overview.jsx

```diff
@@ -1,239 +0,0 @@
-import * as _ from 'lodash-es';
-import * as React from 'react';
-import * as classNames from 'classnames';
-import { Helmet } from 'react-helmet';
-
-import { EventsList } from './events';
-import { SoftwareDetails } from './software-details';
-import { withStartGuide } from './start-guide';
-import { coFetchJSON } from '../co-fetch';
-import { ConsoleHealth, KubernetesHealth, Health } from './graphs/health';
-import { connectToFlags, flagPending } from '../reducers/features';
-import { FLAGS } from '../const';
-import { Gauge, PROMETHEUS_BASE_PATH, requirePrometheus } from './graphs';
-import {
-  AdditionalSupportLinks,
```

Comment:
Nit:  `AdditionalSupportLinks` no longer needs to be exported in utils/documentation.tsx